abb_irb120_moveit_config/config/joint_limits.yaml

```diff
@@ -0,0 +1,31 @@
+joint_limits:
+  joint_1:
+    has_velocity_limits: true
+    max_velocity: 4.36332
+    has_acceleration_limits: false
+    max_acceleration: 0
```

Comment:
I have contacted ABB about getting real acceleration limits, but for the minute I have used the 'old way' of setting them as above.